### ISSUE TITLE
QuantRaPy crashes when not enough tokens to execute call

### ISSUE THREAD

**Issue by BJ1GH** [quantum-sp8de/quantra-installer#1]
would be nice to have normal exit with "Not enough funds to complete task" or similar return
![image](https://github.com/user-attachments/assets/28094ecf-d747-47d0-9413-3226e626af33)

**Comment by plageat** [comment]
Fixed in 1ee5e6c80efafcf5a9

Now on any error from chain output looks like:
```
{
    "code": 500,
    "message": "Internal Service Error",
    "error": {
        "code": 3040000,
        "name": "transaction_exception",
        "what": "Transaction exception",
        "details": [
            {
                "message": "action's authorizing actor 'invalid33333' does not exist",
                "file": "transaction_context.cpp",
                "line_number": 744,
                "method": "validate_referenced_accounts"
            }
        ]
    }
}
```

**Comment by BJ1GH** [comment]
whenever exception occurs running QuantRaPy Ubuntu raises crash warning window asking user either to send report or not 

I would assume there is way to issue warning/error without raising send report window
